openproblems/tasks/chromatin_potential/methods/linear_regression.py

```diff
@@ -31,6 +35,10 @@ def _get_annotation(adata):
         ["pyensembl", "install", "--release", "100", "--species", "mus_musculus"]
     )
     data = EnsemblRelease(100, species="mus_musculus")
```

Comment:
Use `adata.uns["species"]` and `adata.uns["release"]` here instead of hard coded values

Reply:
yes.